### ISSUE TITLE
support query with configuring performance_sensitive in SqlClusterRouter 

### ISSUE THREAD

**Issue by jingchen2222** [4paradigm/OpenMLDB#577]
**Is your feature request related to a problem? Please describe.**

OpenMLDB has supported `performance_sensitive` in cli.
We need to support configure it in SDK (java sdk test or python sdk test).

So we are required to support configuring performance-sensitive in SqlRouterCluster

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by jingchen2222** [comment]
- [x] Support configure performance-sensitive/non-performance-sensitive when query SQL (batch query only)
https://github.com/4paradigm/OpenMLDB/pull/568
```c++
std::shared_ptr<::hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteSQL(const std::string& db, const std::string& sql,
                                                                         ::hybridse::sdk::Status* status,
                                                                         bool performance_sensitive) {
    return ExecuteSQLParameterized(db, sql, std::shared_ptr<openmldb::sdk::SQLRequestRow>(), status,
        performance_sensitive);
}
```


Note: we need to ddd related test cases for cli and SDK query @wuyou10206

**Comment by jingchen2222** [comment]
Closed by #568
